### PR TITLE
Update and Clarify Encryption Guidelines

### PR DESCRIPTION
**Pull request by przadka**
Added details about preferred encryption methods and tools
Provided a link to our VeraCrypt tutorial for more detailed instructions
Clarified the separation between personal and professional digital spaces when using encrypted systems
Added a reminder about responsible use of personal digital spaces on encrypted systems
Clearly defined Docker containers as secure work environments.

### CODE REVIEW

**Comment by mjurbanski-reef** [comment]
please add newlines after `.` https://github.com/reef-technologies/handbook#sembr

File: README.md [hunk 1]
```diff
@@ -63,23 +63,25 @@ UPDATE: A fully encrypted laptop of our senior developer has been stolen from hi
...
+
+You should have an email account within the reef.pl domain. If your role involves dealing with encrypted data, this email address and any associated accounts must be used exclusively within a secure environment, such as a virtual machine or dual-boot system. This ensures a clear segregation between your professional and personal digital spaces, preventing accidental cross-access or data leaks involving customer data. Note that accessing your personal digital spaces (such as private Gmail, Facebook, etc.) is permissible when done from a fully encrypted system, but this should be done responsibly and without compromising work-related security protocols.
+
+Lastly, Docker containers, which store and run client code, are to be considered secure work environments. Accessing these containers from personal systems or environments is prohibited to maintain a robust security structure, preventing any potential cross-access and preserving the integrity of our client's data.
```
Comment (mjurbanski-reef) [inline README.md:84]: > Lastly, Docker containers, which store and run client code, are to be considered secure work environments.

change to 

> Lastly, Docker containers, which store and run client code, are to be considered **secrets**.

**Comment by jsuchan-reef** [comment]
I think it would also deserve to add a paragraph explicitly about (not) using email/Slack on mobile devices.

**Comment by mjurbanski-reef** [comment]
> I think it would also deserve to add a paragraph explicitly about (not) using email/Slack on mobile devices.

sounds like a good idea, let me know if you will create this PR or should I

not sure tho if it belong in encryption section or dedicate one, but I guess we can tackle that problem in dedicated PR

**Comment by jsuchan-reef** [comment]
> sounds like a good idea, let me know if you will create this PR or should I

I dont actually know what the rules are, I was just hoping someone more knowledgeable in this regard will step up, and you did :) I'll gladly leave the PR up to you, thanks.

**Comment by przadka** [comment]
> I dont actually know what the rules are, I was just hoping someone more knowledgeable in this regard will step up, and you did :) I'll gladly leave the PR up to you, thanks.

I am not aware of any no-mobile-devices rule so my guess is that it doesn’t exists.

**Comment by jsuchan-reef** [comment]
> I am not aware of any no-mobile-devices rule so my guess is that it doesn’t exists.


ok. this section states:

> If your role involves dealing with encrypted data, this email address and any associated accounts must be used exclusively within a secure environment, such as a virtual machine or dual-boot system.

So now my question is - is a mobile phone considered "a secure environment (...)" matching the stated criteria? If not, then it shouldnt be allowed to use it on a phone. If phone is considered a secure environment, then I feel there's a slight disparity between what I need to do to use the email on my work machine vs. on my phone.

**Comment by przadka** [comment]
You are right, we should clarify this. It seems this needs a proper S3 discussion rather than a quick handbook PR. I will add an S3 driver.

**Comment by mjurbanski-reef** [comment]
Superseded by #97

**Pull request closed without merging**